service/src/main/resources/db/changesets/20240502_add_imputation_inputs.yaml

```diff
@@ -0,0 +1,114 @@
+# add non-user-provided inputs to the imputation pipeline
+databaseChangeLog:
+  - changeSet:
+      id: add non-user-provided inputs to the imputation pipeline
+      author: mma
+      changes:
+        # add user_provided and default_value columns to the pipeline_input_definitions table
+        - addColumn:
+            tableName: pipeline_input_definitions
```

Comment:
just thought of this while looking at the pr, can we add a unique key constraint on this table for `pipeline_id + name`.  It'll stop us from making possibly silly mistakes in the future for little to no cost.

Reply:
good call!